### DISCUSSION TITLE
Salt not able to perform most commands

### DISCUSSION THREAD

**Discussion by declanobriendc** [Security-Onion-Solutions/securityonion#4260] [Unsupported Versions]
I have a master and 6 sensors. 

I restarted the sensors via sudo salt \* cmd.run 'shutdown -r'.

After the sensors came back up salt fails to issue most commands.

All of the node responses are as follows for all sensors and master:
'so-status' is not available.

**Comment by weslambert** [comment]
Are you still having this issue? Have you tried rebooting these nodes again?
